scripts/fetch_dependencies.sh

```diff
@@ -39,12 +61,14 @@ fi
 
 # All integration types - can be leveraged for local - non CI purposes
 if [[ -z "$INTEGRATION" ]]; then
-  echo "> Integration option was not specified (eg: carthage, cocoapods or manual). Script will grab SDKs using all supported dependency managers."
```

Comment:
No Carthage support for now, right?

Reply:
Yeah, thatll be added once the xcframeworks are in place ;)